.travis.yml

```diff
@@ -5,11 +5,8 @@ jobs:
   fast_finish: true
   include:
    - php: 5.2
-     dist: precise
```

Comment:
These changes will definitely not work as Xenial doesn't support those PHP versions.